src/lib.rs

```diff
@@ -5,7 +5,7 @@
 //! [rust-vmm](https://github.com/rust-vmm/community) components.
 
 #![deny(missing_docs)]
-#[cfg(unix)]
+#[cfg(linux)]
 #[macro_use]
```

Comment:
Should we use "#[cfg(target_os = "linux")]" here?
Seems there's only shortcut for #[cfg("unix")] and no shortcut #[cfg("linux")], which then causes the huge change in code coverage.

Reply:
I wondered about that change in code coverage. The bitflags crate compiles fine on non-Linux, but because it isn't used, it gives a warning "unused `#[macro_use]` import".

Maybe a better solution is to add the shortcut for #[cfg("linux")]?

Reply:
Sorry， I mean "#[cfg(linux)]" will always be false on linux system. We should replace all "#[cfg(linux)]" with "#[cfg(target_os = "linux")]:)